test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -332,6 +332,78 @@ public void OnnxModelMultiInput()
             }
         }
 
+        [OnnxFact]
+        public void OnnxModelOutputDifferentOrder()
+        {
+            var modelFile = Path.Combine(Directory.GetCurrentDirectory(), "twoinput", "twoinput.onnx");
+            var env = new ConsoleEnvironment(seed: 1);
+            var samplevector = GetSampleArrayData();
```

Comment:
Is this variable being used?

Reply:
I'm not sure, let me check and I'll remove it if its not. I copied the test from another one and just changed the output parameters and orders to test my change.

Reply:
Removed it.